app/server/static/components/question_and_answer.vue

```diff
@@ -0,0 +1,78 @@
+<template lang="pug">
+extends ./annotation.pug
+
+block annotation-area
+  div.card
+    header.card-header
+      div.card-header-title.has-background-royalblue
+        div.field.is-grouped.is-grouped-multiline
+          div.control
+            div.tags.has-addons.white(
+              v-if="docs[pageNumber] && docs[pageNumber].extra_text"
+              ) {{ docs[pageNumber].extra_text || 'missing quesiton' }}
```

Comment:
Nit: this should be indented like so:

```pug
            div.tags.has-addons.white(
              v-if="docs[pageNumber] && docs[pageNumber].extra_text"
            ) {{ docs[pageNumber].extra_text || 'missing quesiton' }}
```

Basically there are two extra spaces before the `)`.

Reply:
yep